docs/devel/feature-flow-issue-template.md

```diff
@@ -0,0 +1,63 @@
+Thanks for wanting to add a feature to Kubernetes!  You will be responsible for guiding
+your feature through completion, and asking the right people for approvals.  
+
+Large features typically go through three stages: [Alpha, Beta, and Stable](https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#api-versioning)
+Each stage requires various approvals from various teams.
+
+You can use the checkboxes below to
+track your progress.  You can delete the text above once you have read it.  Please keep the checklist.
+
+- Before Alpha
+    - [ ] Design Approval
+      - [ ] Propose your feature.  See [Advice for proposing features].  **PROPROSAL-ISSUE-NUMBER**
+      - [ ] Start with a < 1 page description of design in an issue comment below this. [advice for design description]  **1PAGER-ISSUE-NUMBER**
+      - [ ] Initial design review  		     [advice for getting a review from a team]  **DESIGN-REVIEW-ISSUE-NUMBER**
+      - [ ] Initial API review (if API) **API-REVIEW-ISSUE-NUMBER**
+        -  cc @kubernetes/api
+    - [ ] Write (code + tests + docs) then get them merged.  **LIST-OF-PRS**
+      - [ ] **Code needs to be disabled by default.**   Verified by code OWNERS
+      - [ ] OWNER approval for code
+      - [ ] Minimal testing
+      - [ ] Minimal docs
+        - *Glossary Section Item* in the docs repo: kubernetes/kubernetes.github.io
+        - cc @kubernetes/docs on docs PR
+        - **cc @kubernetes/feature-reviewers on this issue** to get approval before checking this off
+      - [ ] Update release notes
+- Before Beta
+  - [ ] Testing is sufficient for beta
+  - [ ] User docs with tutorials
+        - *Updated walkthrough / tutorial* in the docs repo: kubernetes/kubernetes.github.io
+        - cc @kubernetes/docs on docs PR
+        - **cc @kubernetes/feature-reviewers on this issue** to get approval before checking this off
+  - [ ] Thorough API review
+    - cc @kubernetes/api
+- Before Stable
+  - [ ] Soak, load testing 			
+  - [ ] detailed user docs and examples
+    - **cc @kubernetes/docs**
```

Comment:
docs/proposals/foo.md needs to move to docs/design/foo.md and be updated to reflect how we did it.  Design docs are required to be maintained separately and in addition to user docs.